### PR TITLE
Feat/hot keys

### PR DESCRIPTION
**Pull request by EdPrys**
Fix: #8 
add: hot-keys

### CODE REVIEW

File: common/utils/hotKeys.ts
Comment (Security2431) [inline common/utils/hotKeys.ts:53]: it scrolls on 14s 🤔 Can you figure out?

File: common/utils/hotKeys.ts
Comment (Security2431) [inline common/utils/hotKeys.ts:49]: it scrolls on 14s 🤔 Can you figure out?

Comment (Security2431) [reply common/utils/hotKeys.ts:49]: Try to scroll at the end video

File: common/utils/hotKeys.ts
Comment (Security2431) [inline common/utils/hotKeys.ts:97]: please add comment what this hot key should do

File: common/utils/hotKeys.ts
Comment (Security2431) [inline common/utils/hotKeys.ts:33]: What this hotkey should do?

File: common/utils/hotKeys.ts
Comment (Security2431) [reply common/utils/hotKeys.ts:49]: At the beginning video it scrolls by 5s, but then by 14s

**Review by Security2431 — CHANGES_REQUESTED** [review]
Looks good, left some comments. Also, please, can you base this PR on main branch? Otherwise, we cannot merge it, before other PRs merged

File: common/utils/hotKeys.ts
Comment (EdPrys) [reply common/utils/hotKeys.ts:33]: turn subtitles on and off

File: common/utils/hotKeys.ts
Comment (Security2431) [inline common/utils/hotKeys.ts:25]:
```suggestion
          // Show/hide subtitles
          if (videoEl instanceof HTMLVideoElement) {
```

File: common/utils/hotKeys.ts
Comment (Security2431) [reply common/utils/hotKeys.ts:55]: Created separate issue #28

File: common/utils/hotKeys.ts
Comment (Security2431) [reply common/utils/hotKeys.ts]: Moved this to separate issue #28

File: components/VideoPlayer/VideoPlayer.tsx
Comment (Security2431) [reply components/VideoPlayer/VideoPlayer.tsx:33]: If you want clarify, feel free to ping me

**Pull request merged**
